specs/online_order_spec.rb

```diff
@@ -1,90 +1,117 @@
 require 'minitest/autorun'
```

Comment:
For this file you've got a lot of similar problems with `order_spec.rb`.  I think fixing these would lead to you  fixing the others.